packages/consent/consent-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ht-sdks/events-sdk-js-consent-tools",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
I decided to manually bump these `package.json` versions to `1.0.0` just to reflect the current major version we're on. This way it's clear that `consent-wrapper-onetrust` has a dependency on `consent-tools@^1.0.0` and going forward we can just use the GitHub release tag to version within the v1 major version.